submissions/LGeorge007/DOM-practice/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
For me as a fan of Heroes of Might and Magic III your work is  🔥 
Please fix the title.

Reply:
I forgot. Fixed.